CefSharp/Internals/JavascriptObjectRepository.cs

```diff
@@ -104,15 +104,33 @@ public bool TryCallMethod(long objectId, string name, object[] parameters, out o
 
             try
             {
-                // Do we have enough arguments? Add Type.Missing for any that we don't have incase of optional params
+                //Fix for C# paramArray and refactor on arguments mismatch 03/22/2016
                 var missingParams = method.ParameterCount - parameters.Length;
-                if (missingParams > 0)
+                if (missingParams > 0 || method.MethodParameters.Any(t => t.IsParamArray))
```

Comment:
Unless I'm mistaken you can only have one `params` per method, and it's the last param, so this can likely be simplified using `Last`